linter.py

```diff
@@ -1,27 +1,8 @@
-#
-# linter.py
-# Linter for SublimeLinter3, a code checking framework for Sublime Text 3
-#
-# Written by Aparajita Fishman
-# Copyright (c) 2015-2016 The SublimeLinter Community
-# Copyright (c) 2013-2014 Aparajita Fishman
-#
-# License: MIT
-#
-
-"""This module exports the PHPLint plugin class."""
-
 from SublimeLinter.lint import Linter
 
 
 class PHPLint(Linter):
-    """Provides an interface to the phplint executable."""
-
-    syntax = ('php', 'html')
-    cmd = 'phpl --php-version 5 --print-path relative --print-column-number --tab-size 4 --no-overall'
-    version_args = '--version'
-    version_re = r'PHPLint (?P<version>\d+\.\d+)'
-    version_requirement = '>= 2.0'
+    cmd = 'phpl --print-path relative --print-column-number --no-overall ${args}'
```

Comment:
Append $file to cmd bc we probably deprecate this automagic stuff in the future.

Reply:
👍 

Question: I see some linters declare cmd as a tuple, is there any benefit to doing it that way?

Reply:
After all we convert to a list, haha. Writing a string looks better and should be okay unless pythons shlex.split fn has a bug.

Reply:
Thanks, good to know. I guess if shlex.split has bugs we’ll notice soon enough 😉